packages/field-base/test/input-control-mixin.test.js

```diff
@@ -369,6 +370,18 @@ const runTests = (defineHelper, baseMixin) => {
         element._preventInputDebouncer.flush();
         expect(element.hasAttribute('input-prevented')).to.be.false;
       });
+
+      it('should check the helper checkbox', async () => {
+        element.allowedCharPattern = '';
+        const helperCheckbox = fixtureSync(`<input id="helper-checkbox" type="checkbox" slot="helper"/>`);
+        element.appendChild(helperCheckbox);
+        helperCheckbox.focus();
```

Comment:
```suggestion
        const checkbox = fixtureSync(`<input type="checkbox" slot="helper"/>`);
        element.appendChild(checkbox);
        checkbox.focus();
```
Can be simplified a bit.